apps/frontend/src/components/Inbox/InboxItems.tsx

```diff
@@ -34,11 +41,26 @@ export const InboxItems: React.FC = () => {
     if (!isFetched) {
       fetchInbox()
     }
-  }, [session, fetchInboxData, setIsFetched])
+  }, [isFetched, fetchInbox])
```

Comment:
Add proper error logging for better debugging like
console.error("Error from fetchInbox::", error)